test/unit/transpiler/passes/scheduling/test_dynamical_decoupling.py

```diff
@@ -165,18 +163,6 @@ def test_insert_dd_ghz_everywhere(self):
         expected = self.ghz4.copy()
         expected = expected.compose(Delay(50), [1], front=True)
 
-        expected = expected.compose(Delay(162), [2], front=True)
-        expected = expected.compose(YGate(), [2], front=True)
-        expected = expected.compose(Delay(326), [2], front=True)
-        expected = expected.compose(YGate(), [2], front=True)
-        expected = expected.compose(Delay(162), [2], front=True)
-
-        expected = expected.compose(Delay(212), [3], front=True)
-        expected = expected.compose(YGate(), [3], front=True)
-        expected = expected.compose(Delay(426), [3], front=True)
-        expected = expected.compose(YGate(), [3], front=True)
-        expected = expected.compose(Delay(212), [3], front=True)
-
```

Comment:
These _shouldn't_ have been DD'd in the first place since the qubits weren't used to this point (ie., they were still in the |0> state). In `main` there is a bug with the dirty qubit handling which this PR fixes.